src/pact-standalone.spec.ts

```diff
@@ -37,146 +37,154 @@ describe('Pact Standalone', function () {
   });
 
   describe('Check if OS specific files are there', () => {
-    describe('OSX', () => {
-      beforeEach(() => (pact = standalone('darwin')));
+    if (!process.env.ONLY_DOWNLOAD_PACT_FOR_WINDOWS) {
```

Comment:
👌

Reply:
Haha! I'd like not to have to do this. But it gets it going for now.